### PR TITLE
Fix(scrollable-grid): styling fixes 

### PR DESCRIPTION
**Pull request by CatalinB7**
- Fixing the grid loading bar jump
- Centering empty state of scrollable grid

### CODE REVIEW

File: projects/angular/components/ui-grid/src/ui-grid.component.ts [hunk 1]
```diff
@@ -274,6 +274,13 @@ export class UiGridComponent<T extends IGridDataEntry>
     @Input()
     selectable = true;
 
+    /**
+     * The property of the column which should receive the deficit width when grid is using scrollable resize strategy.
+     *
+     */
+    @Input()
+    deficitReceivingColumnProperty = '';
```
Comment (anbalase) [inline projects/angular/components/ui-grid/src/ui-grid.component.ts:282]: what happens if I set it to 1st col and try to shrink it? will it work?

**Review by AlexAndriesUiPath — APPROVED** [review]

File: projects/angular/components/ui-grid/src/managers/resize/resize-manager.ts [hunk 2]
```diff
@@ -286,7 +286,7 @@ export abstract class ResizeManager<T extends IGridDataEntry> {
 
     endResize() {
         const entries = [this.current!, this._previous!.neighbour, this._previous!.oppositeNeighbour];
-        this._emitNewColumnPercentages(entries.filter(e => e != null) as IResizeInfo<T>[]);
+        this._emitNewColumnPercentages(entries.filter(e => e != null && !!e.column) as IResizeInfo<T>[]);
```
Comment (gheorghitahurmuz) [inline projects/angular/components/ui-grid/src/managers/resize/resize-manager.ts:289]: nit: `Boolean(e?.column)`

File: projects/angular/components/ui-grid/src/ui-grid.component.ts (hunk 1 above)
Comment (CatalinB7) [reply projects/angular/components/ui-grid/src/ui-grid.component.ts]: removed

File: projects/angular/components/ui-grid/src/managers/resize/resize-manager.ts (hunk 2 above)
Comment (CatalinB7) [reply projects/angular/components/ui-grid/src/managers/resize/resize-manager.ts:289]: https://gist.github.com/arthurvi/66cb1e2bcfc92f99f465e0db04264367#what-do-experts-say

**Review by anbalase — APPROVED** [review]
LGTM

**Pull request merged**